metpy/cbook.py

```diff
@@ -27,4 +39,4 @@ def get_test_data(fname, as_file_obj=True):
     return path
 
 
-__all__ = ('Bunch', 'get_test_data', 'is_string_like', 'iterable')
+__all__ = ('get_test_data', 'iterable')
```

Comment:
Why are we excluding is_string_like here and not making its name visible? Just a design question.

Reply:
Well this is actually a mistake. It's left over from when I was thinking I was just going to remove the function outright. Good catch.